lti.py

```diff
@@ -5,304 +5,304 @@
+        msg = "There was an error editing one of the assignments. (ID: {})"
         msg = msg.format(assignment_id)
         if len(updated_list) > 0:
-            '{} {} assignments have been updated successfully.'.format(
-                msg,
-                len(updated_list)
+            "{} {} assignments have been updated successfully.".format(
+                msg, len(updated_list)
             )
 
         return Response(
-            json.dumps({
-                'error': True,
-                'message': msg,
-                'updated': updated_list,
-            }),
-            mimetype='application/json'
+            json.dumps({"error": True, "message": msg, "updated": updated_list}),
+            mimetype="application/json",
         )
 
     if not request.is_xhr:
-        return render_template('error.htm.j2', message='Non-AJAX requests not allowed.')
+        return render_template("error.htm.j2", message="Non-AJAX requests not allowed.")
 
     try:
         course = canvas.get_course(course_id)
     except CanvasException:
-        msg = 'Error getting course #{}.'.format(course_id)
+        msg = "Error getting course #{}.".format(course_id)
         app.logger.exception(msg)
         return Response(
-            json.dumps({
-                'error': True,
-                'message': msg,
-                'updated': []
-            }),
-            mimetype='application/json'
+            json.dumps({"error": True, "message": msg, "updated": []}),
+            mimetype="application/json",
         )
 
     post_data = request.form
 
-    local_tz = timezone(config.TIME_ZONE)
+    local_tz = timezone(TIME_ZONE)
     assignment_field_map = defaultdict(dict)
 
-    for key, value in post_data.iteritems():
-        if not re.match(r'\d+-[a-z_]+', key):
+    for key, value in six.iteritems(post_data):
+        if not re.match(r"\d+-[a-z_]+", key):
             continue
 
-        assignment_id, field_name = key.split('-')
+        assignment_id, field_name = key.split("-")
         assignment_field_map[assignment_id].update({field_name: value})
 
     if len(assignment_field_map) < 1:
         return Response(
-            json.dumps({
-                'error': True,
-                'message': 'There were no assignments to update.',
-                'updated': []
-            }),
-            mimetype='application/json'
+            json.dumps(
+                {
+                    "error": True,
+                    "message": "There were no assignments to update.",
+                    "updated": [],
+                }
+            ),
+            mimetype="application/json",
         )
 
     updated_list = []
-    for assignment_id, field in assignment_field_map.iteritems():
-        assignment_type = field.get('assignment_type', 'assignment')
-        quiz_id = field.get('quiz_id')
+    for assignment_id, field in six.iteritems(assignment_field_map):
```

Comment:
what changed to use six.iteritems?

Reply:
```python
assignment_field_map.iteritems()
```

became 

```python
six.iteritems(assignment_field_map)
```

Because in Python 3 `iteritems` was replaced by `items`, but `items` in Python 2 is a different operation.

It's just a compatibility thing to make it run in either 2 or 3. Am keeping support for py2 for now until we can do a proper upgrade to 3 later